go.mod

```diff
@@ -292,3 +293,5 @@ require (
 	modernc.org/sqlite v1.14.2-0.20211125151325-d4ed92c0a70f // indirect
 	zombiezen.com/go/sqlite v0.8.0 // indirect
 )
+
+replace github.com/status-im/dev-telemetry => github.com/vpavlin/telemetry v0.0.0-20240807100636-642385d153a3
```

Comment:
This needs to be fixed before merging! Depends on https://github.com/status-im/telemetry/pull/37